### PR TITLE
PW-6613 Add the cartridge [partner]_controllers_changes​

### PR DESCRIPTION
**Pull request by zaiddreakh**

### CODE REVIEW

File: src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+        var createRecurringPaymentAccountResult = AdyenHelper.createRecurringPaymentAccount({
+            Customer: customer
+        });
+
+        if (createRecurringPaymentAccountResult.error) {
+            return false;
+        }
+        pspReference = 'PspReference' in createRecurringPaymentAccountResult && !empty(createRecurringPaymentAccountResult.PspReference) ? createRecurringPaymentAccountResult.PspReference : '';
+        tokenID = 'TokenID' in createRecurringPaymentAccountResult && !empty(createRecurringPaymentAccountResult.TokenID) ? createRecurringPaymentAccountResult.TokenID : '';
+        /*if  (empty(TokenID)  ||  empty(PspReference))  {
```
Comment (maassenbas) [inline src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js:159]: Can we remove the commented code?

File: src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+ * attempts to save the credit card to the customer's payment instruments. If a duplicate card was
+ * detected, the original card is removed after the new card is created. If the card cannot be created
+ * successfully, the transaction is rolled back. Whether successful or not, the paymentinstruments
+ * form is cleared.
+ *
+ * @transaction
+ * @return {boolean} true if the credit card can be verified, false otherwise
+ */
+function create() {
+    if(getAdyenPaymentInstruments()) {
```
Comment (maassenbas) [inline src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js:138]: Should we add the code comments to indicate what is the custom Adyen code?

File: src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/COPlaceOrder.js [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+var PaymentProcessor = app.getModel('PaymentProcessor');
+
+/**
+ * Responsible for payment handling. This function uses PaymentProcessorModel methods to
+ * handle payment processing specific to each payment instrument. It returns an
+ * error if any of the authorizations failed or a payment
+ * instrument is of an unknown payment method. If a payment method has no
+ * payment processor assigned, the payment is accepted as authorized.
+ *
+ * @transactional
```
Comment (maassenbas) [inline src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/COPlaceOrder.js:34]: Can we exclude these folders for the linter?

File: src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js (hunk 2 above)
Comment (zaiddreakh) [reply src/cartridges/adyen_controllers_changes/app_storefront_controllers_changes/cartridge/controllers/PaymentInstruments.js:145]: @maassenbas alright I wrapped the methods that involve changes with our comment tags

**Review by wboereboom — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Pull request merged**